### PR TITLE
Update deps

### LINKED ISSUE #122: Bump poppler to 21.11.0 or over
Poppler 21.11.0 was released on November 1st and fixes a bug we have had for 5 years: https://gitlab.gnome.org/GNOME/gimp/-/issues/1007#note_1314785

This report is for us not to forget. If anyone wants to take care of it, please feel free to propose a MR (@hfiguiere @HarryMichal 😉).

### PR DESCRIPTION
**Pull request by paolostivanin**
* Update GNOME to 41
* Update Poppler to 21.11.0 (fix #122)
* Update poppler-data to 0.4.11
* Update Ghostscript to 9.55.0

### CODE REVIEW

File: org.gimp.GIMP.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "app-id": "org.gimp.GIMP",
     "branch": "stable",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "40",
+    "runtime-version": "41",
```
Comment (hfiguiere) [inline org.gimp.GIMP.json:5]: no. don 't. This require plugins to be updated too.

Comment (paolostivanin) [reply org.gimp.GIMP.json:5]: Ah ok, thanks!

Comment (hfiguiere) [reply org.gimp.GIMP.json:5]: I was only commenting on the runtime version here. I case that wasn't clear.

Comment (paolostivanin) [reply org.gimp.GIMP.json:5]: yep I got that :) I saw that there's another update that's updating deps, that's why I closed this. Once the other will be accepted, I'll update the remaining bits!

**Comment by Jehan** [comment]
PR #123 is now merged. Feel free to propose your additional updates @paolostivanin. 🙂

**Comment by paolostivanin** [comment]
hi, sure thing! you mean including also update to gnome 41? or is this still on hold? :)
thanks

**Comment by Jehan** [comment]
@paolostivanin I'm not sure, let's ask @hfiguiere. Maybe make 2 PRs, one for the deps, one for the runtime. Then we can always update the dependencies first and hold, if needed, the runtime (we'll have to update it at some point too anyway).

**Comment by Jehan** [comment]
@paolostivanin Finally just make a single PR with everything, runtime change included and bump the `org.gimp.GIMP.Plugin` version to 2-41. Do you think you could repost it today or this weekend at latest? There is a security reason (though it may not affect us, and no it's nothing related to log4j 😛).

@hfiguiere If you want to know why, please see here (you must be logged in): https://gitlab.gnome.org/GNOME/gimp/-/issues/7615#note_1336808
Unfortunately Github does not even have security flags.

**Comment by hfiguiere** [comment]
I propose to actually stick to `2-40` as a version branch. If we can still update the plugins and rebuild, so that we avoid the problems outlined with update previously where people have to reinstall their plugins since flatpak doesn't handle that case.

**Comment by Jehan** [comment]
> I propose to actually stick to 2-40 as a version branch.

Sure. You are the boss.

@paolostivanin Listen to @hfiguiere recommendation here, not mine. 👍

**Comment by Jehan** [comment]
Ok so we are releasing today 2.10.30 and I'd want to bump the runtime now for a specific dependency. So I am preparing a build with runtime 41 (kept `2-40` for the plug-in version branch though, as discussed): #126

I have not looked in details to the other deps though, because I want minimum problems in my build for now. So new MR for bumping any dependency is still and always welcome. 🤗

**Comment by Jehan** [comment]
So in the end, I don't bump the runtime because it breaks Webkit building!

What I do instead is making sure that we don't use a second-level library with a vulnerability. We are pretty sure we already don't use it anyway, but just to be double-sure, we don't build poppler (the first-level lib which was using it) with it.

Basically: don't bother my last message. I am still welcoming patches bumping to runtime 41 with webkit still building.

**Pull request closed without merging**